src/yang/ietf-bfd-types.yang

```diff
@@ -105,6 +114,8 @@ module ietf-bfd-types {
   identity bfdv1 {
     base "rt:control-plane-protocol";
     description "BFD protocol version 1 as per RFC 5880.";
```

Comment:
Reword it just say "BFD protocol version 1."

Reply:
Ack.